Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -16,6 +16,7 @@
 import collections
 from subprocess import Popen,PIPE
 import FWCore.ParameterSet.DictTypes as DictTypes
+from FWCore.ParameterSet.OrderedSet import OrderedSet
```

Comment:
These changes are already in the `ConfigBuilder.py`.

Reply:
Won't it just merge then? Let us know if we need to change anything.

Reply:
In principle yes, but the PR would be cleaner, and avoid the `operations` signature, if the PR would be rebased to a (pre)release/IB that contains that commit. Or just drop the first commit without rebasing (it is not immediately clear why it would be needed for this PR even in an older release)

Reply:
I see now! I did not notice that we had picked up that commit somehow. We will rebase without it and force push.

Reply:
@makortel done! Sorry again about that.